script.js

```diff
@@ -1,12 +1,91 @@
-//You can edit ALL of the code here
-function setup() {
-  const allEpisodes = getAllEpisodes();
-  makePageForEpisodes(allEpisodes);
+window.onload = setTimeout(setup, 500);
+
+setTimeout(rootSetup, 500);
+
+// function getAllShows() {
+//   return fetch("https://api.tvmaze.com/shows")
+//     .then((response) => response.json())
+//     .then((data) => {
+//         data.forEach(({name, id}) => {
+//         availableShows.push({name, id});
+//       });
+//     })
+//     .catch((error) => console.log(error));
+// }
+
+// pull all episodes from the selected show
+
+function createShowList() {
+  let showList = [];
+  availableShows.forEach((show) => showList.push(show.name));
+  let select = document.querySelector("#show-selector");
+  showList.sort().forEach((show) => {
+    let option = document.createElement("option");
+    option.innerText = show;
+    select.appendChild(option);
+  });
 }
+setTimeout(createShowList, 300);
+
+// show chosen show
+function showChosen (event) {
+  
+  let chosenShow = availableShows.filter(
```

Comment:
This filtering doesn't seem to be working quite right - are you sure about the conditions you're filtering on?